### PR TITLE
Add qiskit driver

### PR DESCRIPTION
**Pull request by odanielson**
This PR is against the `migrate-to-python3` branch, I'll rebase it on master once that one is merged or closed. It adds support to run a qc program using qiskit. It currently only uses the qiskit simulator.

### CODE REVIEW

File: qp/drivers/qiskit/simulator.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        state.reverse()
+        return state
+
+    for state, count in result.get_counts().items():
+        measurement = measurement_from_str(state)
+        measurement_output = ", ".join(measurement)
+        print(f"Qubit measure: { measurement_output } ({ count } times)")
+
+    # NOTE: This distribution is taken from the executed shots, it's not exactly
+    # the same as the theoretical distribution used in vector and density drivers
```
Comment (DanielJanzon) [inline qp/drivers/qiskit/simulator.py]: That's ok, it must be up to the driver to document exactly how it measures.

File: qp/drivers/qiskit/simulator.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    gatearray.T: QuantumCircuit.t,
+    gatearray.Td: QuantumCircuit.tdg,
+    gatearray.CNOT: QuantumCircuit.cnot,
+    gatearray.I: QuantumCircuit.iden,
+    gatearray.SWAP: QuantumCircuit.swap,
+}
+
+
+def distribution_from_counts(counts, n_qubits):
+    n_states = n_qubits ** 2
```
Comment (DanielJanzon) [inline qp/drivers/qiskit/simulator.py]: This function was a little bit difficult to understand when not having the project freshly in mind - add doc string?

**Review by DanielJanzon — APPROVED** [review]
Cool!

File: qp/drivers/qiskit/simulator.py [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+    circuit.measure(range(num_qbits), range(num_qbits))
+    job = execute(circuit, simulator, shots=num_measures)
+    result = job.result()
+
+    # Draw the circuit
+    print(circuit.draw())
+
+    if print_dist:
+        print(
+            "Final distribution (from shots): ",
```
[the remark this answers is not in the document]
Comment (odanielson) [reply qp/drivers/qiskit/simulator.py]: no, but good improvement :) I'll fix

**Pull request merged**